components/log-viewer-webui/client/src/App.tsx

```
@@ -2,19 +1,87 @@
+
 const App = () => {
+
+    const [show, setShow] = useState(true);
+
     return (
-        <CssVarsProvider modeStorageKey={LOCAL_STORAGE_KEY.THEME}>
-            <QueryStatus/>
-        </CssVarsProvider>
+        <div>
+            <div>
+                <button onClick={() => setShow(!show)}>Toggle</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)